tests/test_storages.py

```diff
@@ -71,6 +73,29 @@ def test_json_readwrite(tmpdir):
     assert get('A short one') is None
 
 
+def test_create_dirs():
+    temp_dir = tempfile.gettempdir()
+    db_dir = ''
+    db_file = ''
+
+    while True:
```

Comment:
@msiemens It will never hang. The loop just generates a temporary directory with the assurance that it does not already exist.